drivers/hmis/lib/form_data/default/records/ce_event.json

```diff
@@ -11,7 +11,27 @@
           "text": "Event Date",
           "mapping": {
             "field_name": "eventDate"
-          }
+          },
+          "bounds": [
+            {
+              "id": "max",
+              "_comment": "cannot be in the future",
+              "type": "MAX",
+              "value_local_constant": "$today"
+            },
+            {
+              "id": "max-exit",
+              "_comment": "cannot be after exit date",
+              "type": "MAX",
+              "value_local_constant": "$exitDate"
+            },
+            {
+              "id": "min",
+              "_comment": "cannot be before entry date",
+              "type": "MIN",
+              "value_local_constant": "$entryDate"
+            }
```

Comment:
I'm probably missing them, but I'm not seeing these restrictions in the dictionary.

Reply:
The "cant be in the future" rule is just common sense, since that day hasn't happened yet.

This might be wrong, but I think Entry/Exit bounds apply to all `Occurrence Point` data elements.

> Occurrence Point/Update – Indicates the element may be
collected and entered at any point **during a project stay** to
track changes over time or document the occurrence of
events (e.g., a service is provided). These types of records
must be able to be entered at any point during the project
stay. Some data elements are collected once per project stay,
but it may fall **at any point during the project stay**. 


I am pretty certain that "project stay" is Entry=>Exit period.

Reply:
Thanks.  That all makes sense.  I just know the CE APR looks for events that occur after the report end, but I think you are correct, the events still need to occur within the "stay".